csv-core/src/lib.rs

```diff
@@ -99,6 +99,8 @@ foo,\"bar,baz\"
 #![deny(missing_docs)]
 #![no_std]
 
+extern crate alloc;
```

Comment:
This is a `no_std` crate and I want it to always be a `no_std` crate. We can't use `alloc` here.

Reply:
fixed